pkg/toplevel/server/toplevel-impl_test.go

```diff
@@ -0,0 +1,117 @@
+// SPDX-FileCopyrightText: 2021-present Open Networking Foundation <info@opennetworking.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+package server
+
+import (
+	externalRef0 "github.com/onosproject/aether-roc-api/pkg/toplevel/types"
+	"github.com/onosproject/onos-api/go/onos/config/admin"
+	v2 "github.com/onosproject/onos-api/go/onos/config/v2"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func Test_convertTrasaction(t *testing.T) {
```

Comment:
this is exactly what is needed. I think if you could give another 1 without some of this data and another 1 with a different set of values and some Path Values